README.md

```diff
@@ -25,9 +25,9 @@ To learn more about SpaceMesh head over to our [wiki](https://github.com/spaceme
 install [Go 1.9.2 or later](https://golang.org/dl/) for your platform
 
 ```
-go get https://github.com/spacemeshos/go-spacemesh
+go get github.com/spacemeshos/go-spacemesh
 ```
-or create this directory `GOPATH$/src/spacemeshos/` and clone the repo into it using:
```

Comment:
good idea - shorter is better.

Reply:
Go tools won't work with that.

For example goimports/gofmt will create import statements:
    import "spacesmeshos/assert"
instead of:
    import "github.com/spacemeshos/go-spacemesh/assert"

Reply:
Good instruction would probably be something like:

- Fork https://github.com/spacemeshos/go-spacemesh project
- Checkout your fork from GitHub
- Move your fork from ($GOPATH/src/github.com/YOURACCOUNT/go-spacemesh) to $GOPATH/src/github.com/spacemeshos/go-spacemesh

That should keep Go tools happy!

Reply:
nice - would you like to updat readme.md?